testing/sdk_security.py

```diff
@@ -159,17 +159,21 @@ def create_service_account(service_account_name: str, service_account_secret: st
     log.info('Remove any existing service account and/or secret')
     delete_service_account(service_account_name, service_account_secret)
 
+    private_key_file = "{}-private-key.pem".format(service_account_name)
```

Comment:
Please instead use a separate `with tempfile.TemporaryDirectory(...)` for each pair, this will also take care of cleaning them up.

Reply:
Thanks @porridge. Yes, in hindsight, that should have been done in the `sdk_security` improvements I pushed through recently.

Reply:
Also, please create a ticket to add these changes to `sdk_security` in `dcos-commons`.

Reply:
OK, will do.

Reply:
@porridge I've added `with tempfile.TemporaryDirectory()`.
@elezar Ticket to port this to dcos-commons: https://jira.mesosphere.com/browse/DCOS-38316